doc/develop/getting_started/index.rst

```diff
@@ -533,6 +533,11 @@ that are used to emulate, flash and debug Zephyr applications.
             sudo cp ~/zephyr-sdk-0.14.2/sysroots/x86_64-pokysdk-linux/usr/share/openocd/contrib/60-openocd.rules /etc/udev/rules.d
             sudo udevadm control --reload
 
+      .. note::
+         Make sure the :envvar:`ZEPHYR_TOOLCHAIN_VARIANT` and :envvar:`ZEPHYR_SDK_INSTALL_DIR` environment variables
```

Comment:
Wait, why? These environment variables are optional these days; the default toolchain variant is `zephyr` if not set, and the installation directory is auto-discovered from a set of standard paths if not set. The "Make sure" language here suggests they are mandatory.

Reply:
I like this edit, because for some projects, the `ZEPHYR_TOOLCHAIN_VARIANT` value was set to something other than `zephyr`, and when installing the new Zephyr toolchain, the old value is left untouched. This means the new install actually doesn't use the new toolchain, but uses whatever toolchain was previously installed. 

I think it is useful to have this mentioned as part of the gsg, because this is (indirectly) where a lot of users will be going to update their toolchain, and they (like me, at first) may not know or remember to even check the `ZEPHYR_TOOLCHAIN_VARIANT`

I agree, however, that the "Make sure" phrasing is misleading.

Perhaps better phrasing would be:

```rst
.. note::
        Make sure the :envvar:`ZEPHYR_TOOLCHAIN_VARIANT` and :envvar:`ZEPHYR_SDK_INSTALL_DIR` environment variables
        If you are installing the Zephyr toolchain to replace an existing Zephyr-compatible toolchain, check
        whether the :envvar:`ZEPHYR_TOOLCHAIN_VARIANT` and :envvar:`ZEPHYR_SDK_INSTALL_DIR` environment variables
        are set.
	
        If they are not set, you may proceed without making any changes (the new Zephyr toolchain will be selected
        by default).

        If they have a value, this will override Zephyr's default toolchain selection, and may prevent the Zephyr
        toolchain from being selected. In this case, either delete them, or set :envvar:`ZEPHYR_TOOLCHAIN_VARIANT`
	to ``zephyr`` and set :envvar:`ZEPHYR_SDK_INSTALL_DIR` to your Zephyr SDK installation directory. This
	will cause the Zephyr toolchain to be selected.
```

Though it is unclear to me what the ``zephyr`` toolchain variant actually does (does it cause the default selection process to occur, or does it specifically choose the Zephyr toolchain? Is "the Zephyr toolchain" even correct terminology, here?

Reply:
@glarsennordic , applied an edited version of your proposal *before* the installation steps. This is because the proposal reads like something you should be aware of before you start installing the SDK. Placing it before also allows to avoid duplicating the content in each tab.

Reply:
@glarsennordic 

The text

```
        If you are installing the Zephyr toolchain to replace an existing Zephyr-compatible toolchain, check
```

seems out of place for a document called the "Getting started guide".

A perennial problem that we face with this document is that everybody wants to include the little bits and pieces that they learned over time when adjusting their environment to fit their use cases, but these end up being noise for people who are really just getting started for the first time. We need to keep fighting to keep this document as short as possible, because installing zephyr is already complex enough.